command.go

```diff
@@ -170,11 +181,15 @@ func (cmd *CommandFunc) configure() {
 	case 0:
 	case 1:
 		if r0 := t.Out(0); r0 != errorType {
-			panic("cli.Command: expected a function returning (error) but got (" + r0.String() + ")")
+			panic(
+				"cli.Command: expected a function returning (error) but got (" + r0.String() + ")",
+			)
 		}
 	case 2:
 		if r0, r1 := t.Out(0), t.Out(1); r0 != intType || r1 != errorType {
-			panic("cli.Command: expected a function returing (int, error) but got (" + r0.String() + ", " + r1.String() + ")")
+			panic(
+				"cli.Command: expected a function returing (int, error) but got (" + r0.String() + ", " + r1.String() + ")",
```

Comment:
Can `r0.String()` or `r1.String()` panic?

Reply:
Reading through the comment, it does not sounds like it could panic? Please feel free to correct me if I am wrong, I am still learning about Go 😅 
![Screen Shot 2022-06-07 at 3 30 26 PM](https://user-images.githubusercontent.com/85896120/172493925-c1f16c35-45fe-45c9-8248-d8f5c7b9faa6.png)

Reply:
Yeah I think you are right. OK.